kolibri/plugins/management/assets/src/vue/modal/index.vue

```diff
@@ -3,9 +3,7 @@
   <!-- Accessibility properties for the overlay -->
 
   <!-- Aria-Hidden and TabIndex in .modal might not be necessary because of conditional rendering -->
-  <!-- mostly there in case we switch to v-show -->
   <div class="modal-overlay"
-    v-if="visible"
     @keydown.esc="closeModal"
     @click="bgClick($event)"
     v-el:modal-overlay
```

Comment:
@radinamatic Should automatically be merged when the PR is merged, but I'll pull anyway.